doc/en/mark.rst

```diff
@@ -52,8 +52,6 @@ should add ``--strict`` to ``addopts``:
         serial
 
 
-.. `marker-iteration`
```

Comment:
Hmm interesting! This should be 

```rst
.. _`marker-iteration`:
```

Nice that the hook catches this.

Reply:
In this case removing it is fine, but in general:

```rst
.. something
``` 

this [is a comment in RST](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#comments), so I'm not sure it is correct to remove

```rst
.. `something`
``` 

is something that should be done automatically.

Reply:
yeah none of this was automatic unfortunately :(

Reply:
Oh I see now, `pygrep` only complains about the matches but doesn't fix them, is that right?

Reply:
correct, it's a cheap way to make a linter